### PR TITLE
fix patch formatting + Issue #41 bug fix

### PR DESCRIPTION
**Pull request by alirezanet**
before:
before

after:
after

### CODE REVIEW

**Comment by alirezanet** [comment]
I've also added -v flag to the apply patch to get more useful information in case of a patch failure.
new output example:
```
Checking patch Server/src/Presentation/appsettings.json...
error: while searching for:
         "System": "Information",
         "Microsoft": "Information",
         "Hangfire": "Warning",
         "HealthChecks.*": "Error",
         "System.Net.Http.HttpClient.health-checks": "Error"
      }
   },

error: patch failed: Server/src/Presentation/appsettings.json:5
error: Server/src/Presentation/appsettings.json: patch does not apply
--- a/Server/src/Presentation/appsettings.json
+++ b/Server/src/Presentation/appsettings.json
@@ -5,7 +5,7 @@
          "System": "Information",
          "Microsoft": "Information",
          "Hangfire": "Warning",
-         "HealthChecks.*": "Error",
+         "HealthChecks.*": "Information",
          "System.Net.Http.HttpClient.health-checks": "Error"
       }
    },
```

File: src/git-istage/Patching.cs [hunk 1]
```diff
@@ -134,14 +133,19 @@ public static string ComputePatch(PatchDocument document, IEnumerable<int> lineI
...
-
-            File.WriteAllText(patchFilePath, patch);
+            var patchLines = patch.Split('\n');
+            
+            // passing -v to git apply will output more useful information in case of a patch failure
+            var arguments = $@"apply -v {cached} {reverse} --whitespace=nowarn ""{patchFilePath}""";
+
+            // Write patch to file with Environment NewLine to ensure line endings are normalized
+            foreach (var pLine in patchLines)
+                File.AppendAllText(patchFilePath,  pLine + Environment.NewLine);
```
Comment (terrajobst) [inline src/git-istage/Patching.cs:147]: Isn't that a longwinded way to say

```C#
File.WriteAllLines(patchFilePath, patchLines);
```

Comment (alirezanet) [reply src/git-istage/Patching.cs:147]: Oops, My bad. I changed this to WriteAllLines.

**Comment by terrajobst** [comment]
Awesome. Thanks!

**Pull request merged**